pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -118,19 +150,40 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	return nil
 }
 
-func (o *MustGatherOptions) resolveMustGatherImage() (string, error) {
-	imageClient, err := v1.NewForConfig(o.Config)
-	if err != nil {
-		return "", err
+func (o *MustGatherOptions) resolveImageStreamTagString(s string) (string, error) {
+	namespace, name, tag := parseImageStreamTagString(s)
+	if len(namespace) == 0 {
+		return "", fmt.Errorf("expected namespace/name:tag")
 	}
-	imageStream, err := imageClient.ImageStreams("openshift").Get("must-gather", metav1.GetOptions{})
+	if len(tag) == 0 {
```

Comment:
This should be not needed `imageutil.SplitImageStreamTag` defaults the tag already.

Reply:
Done